linux/step05_centos6_py27_ius_nginx.sh

```diff
@@ -30,11 +30,16 @@ set -u
 # Install OMERO.web requirements
 file=~omero/OMERO.server/share/web/requirements-py27-nginx.txt
 
-# introduced in 5.2.0
+file=~omero/OMERO.server/share/web/requirements-py27-all.txt
+
+# introduced in 5.3.0
 if [ -f $file ]; then
 	#start-latest
 	/home/omero/omeroenv/bin/pip2.7 install -r $file
 	#end-latest
+elif $(is_less_than $OMEROVER 5.3); then
```

Comment:
:+1: for trying to keep backwards compatibility but I guess there's a question of whether or not we try to do so. Are we basically saying `master` is now 5.3 and therefore we either branch off `dev_5_2` or tell anyone who wants to to use a tag?